docs/cmd/kn_service_update.md

```diff
@@ -41,6 +41,9 @@ kn service update NAME [flags]
   -p, --port int32               The port where application listens on.
       --requests-cpu string      The requested CPU (e.g., 250m).
       --requests-memory string   The requested memory (e.g., 64Mi).
+      --tag strings              Tag revisions, format: --tag revision=tag , example: --tag echo-abcde=current (can be specified multiple times). Use identifier @latest to refer to latest ready revision, for e.g.: --tag LATEST=new (LATEST can be used only once with --tag flag).
```

Comment:
I think the LATEST in the e.g. needs to be udpated to @latest

Reply:
thanks, fixed the typo.